scripts/mods/mod_reforged/dynamic_spawns/parties/bandit_parties.nut

```diff
@@ -6,13 +6,31 @@ local parties = [
 		MovementSpeedMult = 1.0,
 		VisibilityMult = 1.0,
 		VisionMult = 1.0,
+		UpgradeChance = 0.5,
+		IdealSize = 8,
```

Comment:
You cannot define IdealSize and then also overwrite `generateIdealSize` function in the same partydef. Because in this case this defined `IdealSize` is redundant.

In this particular case you should just return `8` in the `generateIdealSize` function if none of the conditions are met e.g. add a final `else` at the end.